<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pantalla_consulta9</class>
 <widget class="QWidget" name="pantalla_consulta9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="header_corridas" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1951</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-bottom: 1px solid #E3E3E3;</string>
   </property>
   <widget class="QPushButton" name="boton_rutas">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size:25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Rutas</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/entrega.png</normaloff>recursos/recursos_empresa/entrega.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_corridas">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Corridas</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/calendario.png</normaloff>recursos/recursos_empresa/calendario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_operadores">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Operadores</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/conductor.png</normaloff>recursos/recursos_empresa/conductor.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_autobuses">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Autobuses</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/autobus.png</normaloff>recursos/recursos_empresa/autobus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_inicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/casa-silueta-negra-sin-puerta.png</normaloff>recursos/recursos_empresa/casa-silueta-negra-sin-puerta.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_reservaciones">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Reservaciones</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/boleto.png</normaloff>recursos/recursos_empresa/boleto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>15</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>recursos/recursos_empresa/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>vista/empresa/recursos/usuario.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_opfiltro_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>1911</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;
border: 2px solid #dcdfe6;
border-radius: 15px;
gridline-color: #e0e0e0;
outline: none;
color: #606266;
border-radius: 15px;
border-bottom: 1px solid #E3E3E3;

</string>
   </property>
   <widget class="QLabel" name="corridas_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>59</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none

</string>
    </property>
    <property name="text">
     <string>Corridas por fecha y ciudad</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_filtroDestinoCorr">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>56</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 12px;
    color: #606266;
    min-height: 10px;
}

QComboBox:hover {
    border-color: #c0c4cc;
}

QComboBox:focus {
    border-color: #409eff;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 4px solid #909399;
}

QComboBox QAbstractItemView {
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    background-color: white;
    outline: none;
    selection-background-color: #409eff;
    selection-color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>TODOS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ensenada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mexicali</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>San Felipe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>San Quintín</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tecate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tijuana</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1571</x>
      <y>21</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border: none;
</string>
    </property>
    <property name="text">
     <string>Fecha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1751</x>
      <y>21</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border: none;</string>
    </property>
    <property name="text">
     <string>Origen</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>56</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit{
	border-radius:5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border: none;</string>
    </property>
    <property name="text">
     <string>Origen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_origen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-radius:5px;
    font-size: 20px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_inicio_2">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>120</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #FF6A36;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #E85F30;   /* Naranja más oscuro */
}

QPushButton:pressed {
    background-color: #CC532A;   /* Más oscuro para click */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/volver.png</normaloff>recursos/recursos_empresa/volver.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="QtableWidget_corridas">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>360</y>
     <width>1050</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background: #ffffff;
    border: 2px solid #e6e8ec;
    border-radius: 8px;
    gridline-color: #f1f1f3;
    selection-background-color: #e8f0fe;
    selection-color: #1a1a1a;
    font-size: 15px;
    color: #333;
    alternate-background-color: #fafbff;
    show-grid: false;
	
}

QTableView::item {
    padding: 8px 12px;
    border: 0px;
	text-align: center;
}

QTableView::item:hover {
    background: #f5f8ff;
}

QTableView::item:selected {
    background: #e2edff;
    color: #1a1a1a;
}

QHeaderView::section {
    background: #ffffff;
    color: #6b7280;
    padding: 10px 12px;
    border: 0;
    border-bottom: 2px solid #e6e8ec;
    font-size: 17px;
    font-weight: 600;
}

QHeaderView::section:hover {
    background: #f7f9fc;
}

QHeaderView {
    background: #ffffff;
    border: 0;
}

QTableView::indicator {
    width: 18px;
    height: 18px;
}

QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}

QScrollBar:vertical {
    background: transparent;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #cfd3da;
    min-height: 25px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background: #b5bac4;
}

QScrollBar::add-line,
QScrollBar::sub-line {
    background: none;
    height: 0px;
}
</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>218</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Salida</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Corrida</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destino</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Autobús</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Matrícula</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Operador</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
